pkg/util/encryption/multi.go

```diff
@@ -0,0 +1,72 @@
+package encryption
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"context"
+
+	"github.com/Azure/ARO-RP/pkg/util/keyvault"
+)
+
+type multi struct {
+	sealer  AEAD
+	openers []AEAD
+}
+
+var _ AEAD = (*multi)(nil)
+
+func NewMulti(ctx context.Context, serviceKeyvault keyvault.Manager, secretName, legacySecretName string) (AEAD, error) {
+	key, err := serviceKeyvault.GetBase64Secret(ctx, secretName, "")
```

Comment:
Does `""` always gets latest and greatest? based on logic it should get latest one we need to use for writes

Reply:
> // secretVersion - the version of the secret. This URI fragment is optional. If not specified, the latest

from `vendor/github.com/Azure/azure-sdk-for-go/services/keyvault/v7.0/keyvault/client.go`